crates/viewer/re_view_spatial/tests/video.rs

```diff
@@ -353,6 +353,11 @@ fn test_video_stream_codec_h264() {
     test_video(VideoType::VideoStream, VideoCodec::H264);
 }
 
+#[test]
+fn test_video_stream_codec_h265() {
+    test_video(VideoType::VideoStream, VideoCodec::H265);
+}
```

Comment:
just below is the same function in commented with a todo, that can be removed now